opentelemetry-otlp/examples/basic-otlp/src/main.rs

```diff
@@ -112,16 +110,14 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
         .init();
 
     let common_scope_attributes = vec![KeyValue::new("scope-key", "scope-value")];
-    let tracer = global::tracer_provider()
-        .tracer_builder("basic")
-        .with_attributes(common_scope_attributes.clone())
-        .build();
-    let meter = global::meter_with_version(
-        "basic",
-        Some("v1.0"),
-        Some("schema_url"),
-        Some(common_scope_attributes.clone()),
+    let library = Arc::new(
+        InstrumentationLibrary::builder("basic")
+            .with_version("1.0")
+            .with_attributes(common_scope_attributes.clone())
+            .build(),
     );
+    let tracer = global::tracer_provider().library_tracer(library.clone());
```

Comment:
wondering why tracing is done slightly differently than metric?

Reply:
implemented `global::library_tracer`